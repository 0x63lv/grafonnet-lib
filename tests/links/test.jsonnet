local grafana = import "grafonnet/grafana.libsonnet";
local links = grafana.links;

{
    basic: links.dashboards("node", ["nodes", "system"]),
    advance: links.dashboards(
        "adv",
        ["advanced"],
        asDropdown=false,
        includeVars=true,
        keepTime=true,
    ),
}
