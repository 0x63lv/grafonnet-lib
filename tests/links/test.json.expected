{
   "advance": {
      "asDropdown": false,
      "icon": "external link",
      "includeVars": true,
      "keepTime": true,
      "tags": [
         "advanced"
      ],
      "title": "adv",
      "type": "dashboards"
   },
   "basic": {
      "asDropdown": true,
      "icon": "external link",
      "includeVars": false,
      "keepTime": false,
      "tags": [
         "nodes",
         "system"
      ],
      "title": "node",
      "type": "dashboards"
   }
}
